src/main/java/mate/academy/springboot/datajpa/exception/ControllerException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springboot.datajpa.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NO_CONTENT)
```

Comment:
[developer.mozilla.org/en-US/docs/Web/HTTP/Status/204](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) WHY HttpStatus.NO_CONTENT? I think status 500 would be enought